### DISCUSSION TITLE
For the Live Demo part

### DISCUSSION THREAD

**Discussion by UG-Asger** [tltneon/lgsl#208] [General]
Hey there tltneon

I have updated the e107 version to 6.2.0/1 on https://urbangamers.dk/LGSL/ (modded version or Darken CSS)and added the SA version (ver. 7 RC, modded Parallax CSS) on https://urbangamers.dk/lgsl7/ if you wanna add them too the list. I only use it for Urban Terror, though.

EDIT: My Discord server is also added

Brg. Asger

**Comment by UG-Asger** [comment]
Maybe edit this: 
![image](https://github.com/user-attachments/assets/3ef2daca-acd3-47a3-9d85-d1b84883c0d6)
It's the e107 version 6.2.1 at my Web Site

Ohhh.. and  https://urbangamers.dk/lgsl7/ is now the latest version and the fixes seems to work well. Good job :)

**Comment by tltneon** [comment]
Oh, my bad. Fixed it!
